src/main/resources/io/jenkins/plugins/designlibrary/EmptyStates/index.jelly

```diff
@@ -0,0 +1,55 @@
+<!--
+The MIT License
+
+Copyright 2022 Jenkins project contributors
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+-->
+
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:s="/lib/samples" xmlns:f="/lib/form" xmlns:l="/lib/layout">
+  <s:layout>
+    <s:section>
+      <s:group>
+        <s:preview>
+            <l:notice title="${%noticeTitle}" icon="symbol-plugins" />
+        </s:preview>
+        <s:code file="empty-states.jelly" />
+      </s:group>
+
+      <p class="jdl-paragraph">${%parameters}</p>
+      <ul class="jdl-list">
+        <li>title - ${%parameters.title}</li>
+        <li>icon - ${%parameters.icon}</li>
+        <li>text - ${%parameters.text}</li>
+      </ul>
+    </s:section>
+
+    <s:section title="${%childSectionTitle}" description="${%descriptionChildContent}">
+      <s:group>
+        <s:preview>
+          <l:notice title="${%childNoticeTitle}" icon="symbol-weather-icon-health-00to19">
+            ${%childNotionContent}

```

Comment:
if you're including content like this I think it would make for sense to include some HTML like maybe a call to action.

e.g. a button.

There's a (complex) example here:

https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/jenkins/security/ApiTokenProperty/config.jelly#L104-L106

I would suggest something like (adjust the formatting to something that looks good in the example)::

```suggestion
            <button 
              type="button"
              class="jenkins-button jenkins-button--secondary">${%New item}</button>

```